src/app/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+add_library(clio_app)
+target_sources(clio_app PRIVATE CliArgs.cpp ClioApplication.cpp)
+
+target_link_libraries(clio_app PUBLIC clio_etl clio_feed clio_web clio_rpc)
+
+# target_compile_features(clio_app PUBLIC cxx_std_23)
```

Comment:
I assume we don't need this line?

Reply:
I don't see it added anywhere else. Please check we really don't need it

Reply:
It is defined for `clio_options` library, so I think here it was duplicated.